tfx/utils/io_utils.py

```diff
@@ -100,8 +100,7 @@ def write_string_file(file_name: Text, string_value: Text) -> None:
   """Writes a string to file."""
 
   fileio.makedirs(os.path.dirname(file_name))
-  with fileio.open(file_name, 'w') as f:
-    f.write(string_value)
+  fileio.open(file_name, 'w').write(string_value)
```

Comment:
this is going to reintroduce this [issue](https://github.com/tensorflow/tfx/issues/3265), which was fixed by this [PR](https://github.com/tensorflow/tfx/pull/3279)